From adbde4e97348e1b876d8b3712677d6ae58b45466 Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Thu, 1 Dec 2016 07:20:52 -0500
Subject: [PATCH 2/2] setup-uboot-env-k2g-evm.sh: update for latest default env

* We now need to call "findfdt" so that "name_fdt" is defined.

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 setup-uboot-env-k2g-evm.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup-uboot-env-k2g-evm.sh b/setup-uboot-env-k2g-evm.sh
index f41d278..c80e506 100755
--- a/setup-uboot-env-k2g-evm.sh
+++ b/setup-uboot-env-k2g-evm.sh
@@ -187,7 +187,7 @@ do_expect "\"$prompt\"" "send \"setenv name_ubi $ubifsimage\"" $cwd/setupBoard.m
 do_expect "\"$prompt\"" "send \"setenv name_kern $kernelimage\"" $cwd/setupBoard.minicom
 do_expect "\"$prompt\"" "send \"setenv name_uboot $ubootimage\"" $cwd/setupBoard.minicom
 
-do_expect "\"$prompt\"" "send \"setenv bootcmd 'run envboot; run init_\${fs_mode} get_pmmc_\${kern_mode} run_pmmc get_fdt_\${kern_mode} get_mon_\${kern_mode} get_kern_\${kern_mode} run_mon run_kern'\"" $cwd/setupBoard.minicom
+do_expect "\"$prompt\"" "send \"setenv bootcmd 'run envboot; run init_\${fs_mode} get_pmmc_\${kern_mode} run_pmmc get_mon_\${kern_mode} run_mon findfdt get_fdt_\${kern_mode} get_kern_\${kern_mode} run_kern'\"" $cwd/setupBoard.minicom
 
 do_expect "\"$prompt\"" "send \"saveenv\"" $cwd/setupBoard.minicom
 
-- 
1.9.1

